SU2_CFD/src/output/CMultizoneOutput.cpp

```diff
@@ -173,7 +173,7 @@ void CMultizoneOutput::SetMultizoneHistoryOutputFields(const COutput* const* out
 
       /*--- Determine whether Maker_Analyze/Monitoring has to be used. ---*/
       auto* Marker = &Marker_Monitoring;
-      if (group == "FLOW_COEFF_SURF")
+      if ((group == "FLOW_COEFF_SURF") || (group == "SPECIES_COEFF_SURF"))
```

Comment:
I am now adding species_coeff_surf to the history data. I think this line is sufficient...